brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3300 [Buttons] By default buttons alignment should be left and have backward compatibility for old block.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3300

### Screenshots

- https://d.pr/v/h18kd5

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Switch to master
- Drag and drop buttons
- Set different alignment in master like Center, left, right and full.
- Now switch to branch and make sure after refresh the editor it will not break anything.
- Drag and drop new buttons and make sure it align to left by default and only one buttons should be visible.

### Checklist:
